roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -17,36 +17,33 @@ def run_hammer_cmd(command)
 
 def get_info_from_hammer(command, column=1)
   bash_parse = " | grep -v \"Warning:\" | tail -n+2 | awk -F, {'print $#{column}'}"
```

Comment:
I only now read this and I already wondered why it wasn't needed to use `--columns` in the hammer command. You can drop the tail command by using `--no-headers` and I think to avoid `awk` you can use `--columns` (but I don't see it right now in `--help`). However, separate issue.

Reply:
I know it's beyond the original scope of this PR, but I found an elegant way to solve it with just a few lines of change, using `hammer --output=json ...` instead of `hammer --csv ...`

BTW, I appreciate your insights and observations in these reviews -- they have been very valuable to me.

Reply:
Another note about it, `--no-headers` wasn't necessary when using `hammer --output=json`

But looking over my work, I realized I did remove the piece that filters warnings. I'm not immediately sure whether those are stdout or stderr with hammer (I would assume the latter, but haven't checked) but without filtering, they would definitely break the JSON parsing :/

This all makes me wonder whether it's better to use the API directly to bypass the shell, or if it's better to rely on hammer as the API client in case the API changes in the future (although hammer itself can also change, so I don't know)

Also there should probably be some rescue for `JSON::ParserError`

If I don't think of an obvious solution soon, I'll probably split that piece off into a separate PR